code-rules.tex

```diff
@@ -283,7 +285,7 @@ \section*{Tip 3: Decide if you are ending, pausing, or handing off}
 
 However,
 fiscal sponsorship is only an option in some countries,
-and being accepted as a sponsored project is a long and difficult process.
+and being accepted as a sponsored project can be a long and difficult process.
```

Comment:
(acceptance to https://opencollective.com/opensource can be automatic if your project meets certain preconditions)

Reply:
This is the criteria: https://docs.oscollective.org/getting-started/acceptance-criteria 
(I still argue that complying with those criteria is not easy ;-))

Reply:
Then I will suggest to use "most of the time is". 
In my experience has been like that with several projects.